src/checkbox/src/Checkbox.js

```diff
@@ -60,7 +60,7 @@ const Checkbox = memo(
       }
     }, [ref, indeterminate])
 
-    const themedClassName = useCheckboxAppearance(appearance)
+    const { className: themedClassName, boxProps } = useCheckboxAppearance({ appearance })
```

Comment:
This will become
```suggestion
    const { className: themedClassName, ...styleProps } = useStyleConfig(
      'Checkbox',
      { appearance },
      pseudoSelectors,
      internalStyles
    )
```